OpenRA.Mods.Common/Traits/Cargo.cs

```diff
@@ -207,7 +207,7 @@ public string VoicePhraseForOrder(Actor self, Order order)
 			if (order.OrderString != "Unload" || IsEmpty(self))
 				return null;
 
-			return self.HasVoice("Unload") ? "Unload" : "Move";
+			return self.TraitsImplementing<IVoiced>().FirstOrDefault().HasVoice(self, "Unload") ? "Unload" : "Move";
```

Comment:
Please change this to
```
return self.TraitsImplementing<IVoiced>().Any(x => x.HasVoice(self, "Unload")) ? "Unload" : "Move";
```
`.FirstOrDefault().HasVoice()` is a potential NullReferenceException.